register/murmur3/multihash_murmur3.go

```diff
@@ -0,0 +1,38 @@
+/*
+	This package has no purpose except to perform registration of multihashes.
+
+	It is meant to be used as a side-effecting import, e.g.
+
+		import (
+			_ "github.com/multiformats/go-multihash/register/murmur3"
+		)
+
+	This package registers multihashes for the murmur3 family.
+*/
+package murmur3
+
+// import (
+// 	"hash"
+//
+// 	"github.com/gxed/hashland/murmur3"
+//
+// 	"github.com/multiformats/go-multihash"
+// )
+
+func init() {
+	// REVIEW: what go-multihash has done historically is New32, but this doesn't match what the multihash table says, which is 128!  Resolution needed.
+	// REVIEW: I have also heard that something in ipfs unixfsv1 uses a murmur hash, but that is yet different than this.  Resolution needed.
```

Comment:
Sharded directories do, but I don't think they use go-multihash. However, they _do_ use the codec.

Reply:
See https://github.com/multiformats/go-multihash/issues/135#issuecomment-791178958

I'm pretty sure that we should just remove murmur3 from this library until we can figure out what the correct implementation of the relevant code number is/should be.

Reply:
Adin's made some more comments about this in https://github.com/multiformats/go-multihash/issues/135 as well.

My (weakly held) position is that we should just flat out remove this.  It sounds like anything currently attempting to use it will get answers that are "wrong" already, so we dearly hope nothing is (or we have data inertia problems coming, which are much _bigger_ problems than a library API break) -- at such a point, I have a hard time seeing any good that can come of its inclusion.

If we do want to keep it, I won't object, but I somebody needs to tell me exactly what code goes here.  I will probably "popcorn" this PR to them to add whatever that diff is.

Reply:
Removing it sounds reasonable.

Reply:
Sold.